pkgs/misc/tmux-plugins/tmux-thumbs/default.nix

```
@@ -0,0 +1,28 @@
+{ pkgs, lib, mkTmuxPlugin, fetchFromGitHub }:
```

```suggestion
{ lib, mkTmuxPlugin, fetchFromGitHub, thumbs }:
```
You can directly import thumbs